tests/integration/targets/ipam_subnet/tasks/main.yml

```diff
@@ -284,27 +284,7 @@
         that:
           - subnet is not failed
           - subnet_info.objects | length == 1
-          - subnet_info.objects[0].ddns_send_updates == false
-
-    - name: "Create a Subnet with ddns_ttl_percent set to 25"
```

Comment:
Instead of deleting the Task , it would be a better idea to comment it out and add the reason as well as the ticket number indicating why the task is skipped

Reply:
done